core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/meta/MetaExpression.kt

```diff
@@ -11,19 +11,26 @@ import ch.ergon.dope.validtype.NumberType
 import ch.ergon.dope.validtype.StringType
 import ch.ergon.dope.validtype.ValidType
 
-class MetaExpression(private val bucket: Bucket) : TypeExpression<StringType>, FunctionOperator {
+class MetaExpression(private val bucket: Bucket?) : TypeExpression<StringType>, FunctionOperator {
     override fun toDopeQuery(): DopeQuery {
-        val bucketDopeQuery = bucket.toDopeQuery()
-        return DopeQuery(
-            queryString = toFunctionQueryString(
-                symbol = "META",
-                when (bucket) {
-                    is AliasedBucket -> "`${bucket.alias}`"
-                    is UnaliasedBucket -> bucketDopeQuery.queryString
-                },
-            ),
-            parameters = bucketDopeQuery.parameters,
-        )
+        if (bucket == null) {
+            return DopeQuery(
+                queryString = "META()",
```

Comment:
Since META is now used in two places, please extract it as a constant (like e.g. the ORDER_BY-constant)

Reply:
done